apps/web/src/components/providers/AuthProvider.tsx

```diff
@@ -32,6 +32,7 @@ type OrganizationState =
 
 export type AuthContextValue = UserState &
   OrganizationState & {
+    shouldMonitor: boolean;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** The name `shouldMonitor` is lacking a bit of description for me right now - what we we monitoring? Can we make it `shouldMonitor<Something>`? ‏

Reply:
This value is now only used in the `useMonitoring()` hook that based on the value identifies users/orgs with segment/launchDarkly.

In reality I don't expect this value to be used anywhere else, as it is false only in one case (when a user is in the process of signing up), so I could move the logic for its calculation directly in the hook that uses it, wdyt?

Reply:
I agree, moving the logic to be co-located with it's consumer makes the most sense given that the use is limited.

Reply:
addressed